pkg/ds/farm_test.go

```diff
@@ -268,22 +293,30 @@ func TestContendSelectors(t *testing.T) {
 
 	anotherManifestBuilder := manifest.NewBuilder()
 	anotherManifestBuilder.SetID(anotherPodID)
-	anotherPodManifest := manifestBuilder.GetManifest()
+	anotherPodManifest := anotherManifestBuilder.GetManifest()
```

Comment:
this is probably the same class of bug. we were checking for the other pod manifest before but it will pass if you check fast enough